core-js/src/main/javascript/cdf-core-require-js-cfg.js

```diff
@@ -30,7 +30,7 @@
     requireCfg.paths['cdf'] = ENVIRONMENT_CONFIG.paths["cdf"];
 
   } else if (typeof KARMA_RUN !== "undefined") { // unit tests
-    requireCfg.paths['cdf'] = 'target/test-javascript/cdf';
+    requireCfg.paths['cdf'] = 'target/test-javascript/cdf/cdf';
```

Comment:
Using the assembly now places all code to test under a `cdf` folder, and it was not possible to "expand" the files of the `dir` format while ignoring the root folder.